tests/integration_tests/security/demo_seccomp/src/bin/seccomp_rules/mod.rs

```diff
@@ -20,14 +20,18 @@ pub fn jailer_required_rules() -> Vec<SyscallRuleSet> {
         allow_syscall(libc::SYS_rt_sigprocmask),
         allow_syscall(libc::SYS_rt_sigaction),
         allow_syscall(libc::SYS_execve),
-#[cfg(target_arch = "x86_64")]
+        #[cfg(target_arch = "x86_64")]
         allow_syscall(libc::SYS_mmap),
-#[cfg(target_arch = "x86_64")]
+        #[cfg(target_arch = "aarch64")]
+        // See this issue for why we are doing it this way on arch64:
+        // https://github.com/rust-lang/libc/issues/1348.
```

Comment:
Can we open an issue about this in firecracker-microvm and submit a PR to libc? It looks like an easy fix and we wouldn't need to have hardcoded values in the code.

In the meantime, please specify in the comment what syscall is this one that you're whitelisting.